src/viur/core/bones/json.py

```diff
@@ -46,7 +46,10 @@ def singleValueSerialize(self, value, skel: 'SkeletonInstance', name: str, paren
         return utils.json.dumps(skel.accessedValues[name])
 
     def singleValueUnserialize(self, val):
-        return utils.json.loads(val)
+        if isinstance(val, (str, bytes, bytearray)):
+            return utils.json.loads(val)
+
+        return utils.json.loads(utils.json.dumps(val))
```

Comment:
Hmm this looks a little crazy... doesn't just
```suggestion
        return val
```
work?

Reply:
Can you anyway give a short hint in form of a comment why it must be done this way?